website/content/partials/components/platform-kubernetes.mdx

```diff
@@ -103,7 +135,7 @@ If the replica count is maintained outside waypoint, for instance by a pod autos
 
 A map of resource limits and requests to apply to a pod on deploy.
 
-Resource limits and requests for a pod. limits and requests options must start with either 'limits_' or 'requests_'. Any other options will be ignored.
+Resource limits and requests for a pod. limits and requests options must start with either 'limits*' or 'requests*'. Any other options will be ignored.
```

Comment:
Uh oh.... should we have a way to escape these so the auto gen docs don't change them?